src/explore-education-statistics-common/src/modules/table-tool/components/TableToolWizard.tsx

```diff
@@ -68,11 +68,16 @@ export interface TableToolWizardProps {
   initialState?: Partial<InitialTableToolState>;
   hidePublicationSelectionStage?: boolean;
   finalStep?: (props: FinalStepRenderProps) => ReactElement;
+  loadingFastTrack?: boolean;
   renderFeaturedTable?: (featuredTable: FeaturedTable) => ReactNode;
   scrollOnMount?: boolean;
+  tableSizeErrorDownloadAvailable?: boolean;
+  tableSizeErrorLogEvent?: (
```

Comment:
Would make name changes here based on comments in `FiltersForm` too

Reply:
Happy that this has been addressed :)